cmake/onnxruntime_hosting.cmake

```diff
@@ -46,7 +46,7 @@ target_include_directories(${PROJECT_NAME} PRIVATE
         PUBLIC
         ${Boost_INCLUDE_DIR}
         ${re2_src}
-        ${ONNXRUNTIME_ROOT}/hosting/include
+        ${ONNXRUNTIME_ROOT}/hosting/server
```

Comment:
I would prefer calling it `http` because we are going to have `grpc` later.

Reply:
I like it. Done.